beta/nncf/tensorflow/initialization.py

```diff
@@ -0,0 +1,59 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Union
+
+import tensorflow as tf
+
+from nncf.common.initialization import NNCFDataLoader
+from nncf.config.structure import BNAdaptationInitArgs
+from nncf.config import NNCFConfig
+
+
+class InitializingDataLoader(NNCFDataLoader):

```

Comment:
Seems like this class is general for all algorithms which should use dataset. I suggest using the `TFNNCFDataLoader` name.
@alexsu52 what do you think?

Reply:
I suggest renaming `InitializingDataLoader` to `TFInitializingDataLoader` for parity with Torch backend and respect to the style guide.

Reply:
Ranamed